main.cc

```diff
@@ -2129,6 +2129,10 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
                 ss.local().drain_on_shutdown().get();
             });
 
+            auth_service.invoke_on_all([] (auth::service& auth_service) -> future<> {
```

Comment:
I think I commented on that before, why check all shards if only shard 0 is responsible for creation?
I guess startup is done on shard 0 so it's just enough to do:
```
bool ensure_superuser_is_created() {
SCYLLA_ASSERT(this_shard_id() == 0);
return ...
};

...
void main() {
...
co_await auth_service.local().ensure_superuser_is_created();
...
}
```